compilation error from bad `MK_BOOTLOADER_VERSION`

**Describe the bug**
When Compiling any board the build fails when compiling main.c, see the error below.

**Set up**
  - Board: feather_nrf52840_express and a custom board (testing express to verify build works correctly)
  - OS: A rather fresh install of macos 10.15.6 running python 3.8  

**To Reproduce**
Steps to reproduce the behavior:
1. clone the repo on master
2. clone all submodules recursively 
3. install dependencies 
4. run ` make BOARD=feather_nrf52840_express all` or ` make BOARD=feather_nrf52840_express` with and without cleaning first
5. See error

**Expected behavior**
normal build cycle, run without major errors.

**The Error**
```
CC dfu_ble_svc.c
CC dfu_init.c
CC flash_nrf5x.c
CC main.c
src/main.c: In function 'main':
<command-line>:0:25: error: expected expression before '<<' token
src/main.c:184:34: note: in expansion of macro 'MK_BOOTLOADER_VERSION'
   BOOTLOADER_VERSION_REGISTER = (MK_BOOTLOADER_VERSION);
                                  ^~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
make: *** [_build/build-feather_nrf52840_express/main.o] Error 1
```

**conjecture**
when searching the whole repo (including the submodules) there are only 3 mentions of `MK_BOOTLOADER_VERSION`:
- makefile line 290: `CFLAGS += -DMK_BOOTLOADER_VERSION='($(word 1,$(_VER)) << 16) + ($(word 2,$(_VER)) << 8) + $(word 3,$(_VER))'`
- main.c line 184: `  BOOTLOADER_VERSION_REGISTER = (MK_BOOTLOADER_VERSION);`
- an .emPower file ?

I think the issue is coming from line 290 in the make file. To me, it looks like this line is adding a preprocessor macro through the command line (see link 1) to `#define` the bootloader version number. To me the specified expression looks fine, for either python or c, I can only assume there is some compounding error?

additionally, in issue 159 in this repo 'ahtashambaig' had a similar issue. however, it seems that his issue stemmed from a windows command name issue. 

any suggestions to fix it?

link 1: research about `-D`: https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html

--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ MBR_HEX			 = lib/softdevice/mbr/hex/mbr_nrf52_2.4.1_mbr.hex
 # linker by MCU eg. nrf52840.ld
 LD_FILE      = linker/$(MCU_SUB_VARIANT).ld
 
-GIT_VERSION != git describe --dirty --always --tags
+GIT_VERSION := $(shell git describe --dirty --always --tags)
 GIT_SUBMODULE_VERSIONS != git submodule status | cut -d" " -f3,4 | paste -s -d" " -
 
 # compiled file name

